src/nycdb/sql/dob_certificate_occupancy.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX dob_certificate_occupancy_bbl_idx on dob_certificate_occupancy (bbl);
```

Comment:
Since we are keeping only `binnumber`, can we rename it in the sql here to just `bin` for consistency with other datasets - and add indexes for `bin`, `jobnumber` as well?